docs/docsearch-scraper-config.json

```diff
@@ -0,0 +1,63 @@
+{
+  "index_name": "dev_docs",
+  "start_urls": [
+    {
+      "url": "https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/",
+      "selectors_key": "folder",
+      "page_rank": 0,
+      "tags": [
+        "folder"
+      ]
+    },
+    {
+      "url": "https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/.+$",
+      "selectors_key": "folder",
+      "page_rank": 0,
+      "tags": [
+        "folder"
+      ]
+    },
+    {
+      "url": "https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/.+\\.md$",
+      "selectors_key": "file",
+      "page_rank": 5,
+      "tags": [
+        "file"
+      ]
```

Comment:
And finally, this tells the scraper that these files—markdown files in the `va.gov-team` repo—should use the `selectors_key` of `file`.

With all these `start_urls`, to add more repos, we can [add a variable](https://docsearch.algolia.com/docs/config-file/#using-regular-expressions) to the `url` for the repo name.

Any URL that doesn't fit these `start_urls` will have no `selectors_key`. Because `selectors` below doesn't have a `"default"` property, they won't be crawled or indexed.